dll/win32/comdlg32/filedlg.c

```diff
@@ -3644,7 +3644,11 @@ void FILEDLG95_FILENAME_FillFromSelection (HWND hwnd)
  * Although shell32 is already linked the behaviour of exported StrRetToStrN
  * is dependent on whether emulated OS is unicode or not.
  */
+#ifdef __REACTOS__
+static HRESULT FILEDLG_StrRetToStrNW (LPWSTR dest, DWORD len, LPSTRRET src, const ITEMIDLIST *pidl)
```

Comment:
I don't understand why they would duplicate this, instead of sharing it...
Anyway, this is a workaround for now.